build.gradle

```diff
@@ -13,14 +13,18 @@ buildscript {
 
 //add dependency repositories
 repositories {
-	maven {
-		name = "CB Repo"
-		url = "http://chickenbones.net/maven"
-	}
-	maven {
-		name = "JEI Maven"
-		url = "http://dvs1.progwml6.com/files/maven"
-	}
+    maven {
+        name = "CB Repo"
+        url = "http://chickenbones.net/maven"
+    }
+    maven {
+        name = "JEI Maven"
+        url = "http://dvs1.progwml6.com/files/maven"
+    }
+    maven {
+        name = "CurseForge"
+        url = "https://minecraft.curseforge.com/api/maven/"
+    }
```

Comment:
I honestly don't think its an issue, and i only use FG to setup forge and actually build jars for deployment. Also, all the mods listed there except for AE and BuildCraft i contribute directly to so that isn't an issue for me. Anyway, back to the subject, i would still like the sources placed in /src/api/java.